### PR TITLE
Environment Config

### PR DESCRIPTION
**Pull request by mariaefOT**
Test API and environment.

### CODE REVIEW

File: src/components/prueba.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
+import axios from 'axios';
+import React, {useEffect, useState} from 'react';
```
Comment (ghost) [inline src/components/prueba.js]: You don't need to `import React from 'react'` in the latest React versions.

File: src/components/prueba.js [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+const Prueba = () => {
+const [token, setToken] = useState([]);
```
Comment (ghost) [inline src/components/prueba.js]: I'd set the initial state to an empty string because the token is a string.

File: src/components/prueba.js [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+const [res, setRes] = useState([]);
```
Comment (ghost) [inline src/components/prueba.js]: Even though this is just a test, I'd name this state differently. Example: accounts.

File: src/components/prueba.js [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+
+const apiToken = () => {
```
Comment (ghost) [inline src/components/prueba.js]: This function should have a name similar to the endpoint. Example: _authenticateUser_ or just _authenticate_.

File: src/components/prueba.js [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+    })
+    .then(response => {
+        setToken(response.data.authToken);
+        console.log(token);
+    }).catch(error => {
+        console.log(error);
+    })
+};
+
+const apiGetAccount = () => {
```
Comment (ghost) [inline src/components/prueba.js]: getAccount**s** sounds better to me!

**Review by ghost — CHANGES_REQUESTED** [review]
Files created for the sake of holding components should have a name that follows the PascalCase naming convention.

File: src/components/prueba.js (hunk 1 above)
Comment (mariaefOT) [reply src/components/prueba.js]: Thanks, I didn't know that. I'll keep that in mind from now on.

**Comment by mariaefOT** [comment]
Thanks. Make the suggested changes. I wait for comments to merge.

File: src/components/prueba.js (hunk 1 above)
Comment (ghost) [reply src/components/prueba.js]: No problem! That's the main objective of code reviews; learning from each other! :)

**Review by ghost — APPROVED** [review]

**Pull request merged**